include/aspect/compat.h

```diff
@@ -30,53 +30,7 @@
 namespace big_mpi
 {
 
-#if DEAL_II_VERSION_GTE(9,4,0)
-
-  using dealii::Utilities::MPI::broadcast;
-
-#else
-
-  inline MPI_Datatype
-  mpi_type_id(const char *)
-  {
-    return MPI_CHAR;
-  }
-
-  /**
-   * Broadcast the information in @p buffer from @p root to all
-   * other ranks.
-   */
-  template <typename T>
-  void
-  broadcast(T                 *buffer,
-            const size_t       count,
-            const unsigned int root,
-            const MPI_Comm    &comm)
-  {
-    Assert(root < dealii::Utilities::MPI::n_mpi_processes(comm),
-           dealii::ExcMessage("Invalid root rank specified."));
-
-    // MPI_Bcast's count is a signed int, so send at most 2^31 in each
-    // iteration:
-    const size_t max_send_count = std::numeric_limits<signed int>::max();
-
-    size_t total_sent_count = 0;
-    while (total_sent_count < count)
-      {
-        const size_t current_count =
-          std::min(count - total_sent_count, max_send_count);
-
-        const int ierr = MPI_Bcast(buffer + total_sent_count,
-                                   current_count,
-                                   mpi_type_id(buffer),
-                                   root,
-                                   comm);
-        AssertThrowMPI(ierr);
-        total_sent_count += current_count;
-      }
-
-  }
-#endif
+using dealii::Utilities::MPI::broadcast;
```

Comment:
This is correct, but not correctly indented. github is probably going to ask you to indent this as it was before, i.e., by two spaces.

Reply:
Sure, Thanks